---
layout: default
---
# GB0270

## Graph-Break Type
*Short name describing what triggered the graph break*

unimplemented builtin op vars() with no arguments

## Context
*Values or code snippet captured at the break point*

vars: {self} {args}

## Explanation
*Explanation of why the graph break was triggered*

Dynamo does not know how to trace builtin operator {self.fn} with no arguments

## Hints
*Hints on how to resolve the graph break*

- It may be possible to write Dynamo tracing rules for this code. Please report an issue to PyTorch if you encounter this graph break often and it is causing performance issues.


## Additional Information

<!-- ADDITIONAL INFORMATION START - Add custom information below this line -->

<!-- ADDITIONAL INFORMATION END -->


[Click here to add Additional Info](https://github.com/meta-pytorch/compile-graph-break-site/edit/main/docs/gb/gb0270.md)

[Back to Registry](../index.html)
